### ISSUE TITLE
Use ':editor' on freecodecamp.com

### ISSUE THREAD

**Issue by JacobTravers** [tridactyl/tridactyl#3851]
Hello I wanted to see if there is any way to make the editor command work on [freecodecamp.com](https://freecodecamp.com), I noticed it can extract the text from the editor on the page but it wont inject your temp-file once you write and quit. This would be a GREAT feature as I use [kakoune](https://kakoune.com) as my editor and HATE the builtin editor on FreeCodeCamp.com .

**Comment by bovine3dom** [comment]
Duplicate of https://github.com/tridactyl/tridactyl/issues/1267 :)

It shouldn't be a very difficult fix if you're interested in having a go.

**Comment by JacobTravers** [comment]
@bovine3dom this fix doesn't appear to be working on freecodecamp.com. Testing with firefox on arch as well as the beta version of tridactyl.

**Comment by bovine3dom** [comment]
Ah, sorry, I thought freecodecamp used CodeMirror but they use Monaco instead: https://microsoft.github.io/monaco-editor/api/modules/monaco.editor.html

It probably wouldn't take long to do something similar to what we did with freecodecamp.

Thanks for pointing it out : )

**Comment by bovine3dom** [comment]
E.g. https://github.com/glacambre/firenvim/blob/35b46e75a2c39240854f4b3f1056fd6cc75617a4/src/editors/MonacoEditor.ts

**Comment by bovine3dom** [comment]
Fixed for real this time in #3888

**Comment by glacambre** [comment]
>Fixed for real this time in #3888

I don't think so, editor-adapter doesn't work (anymore) on FreeCodeCamp (Firenvim issue: https://github.com/glacambre/firenvim/issues/978 ) :(

**Comment by bovine3dom** [comment]
Eugh, I tested it on Monaco and assumed that it would work in free code camp.

Thanks :(

**Comment by dqgthb** [comment]
Wondering if there is any update on this issue.
